e2e/playwright/file-tree.spec.ts

```diff
@@ -345,12 +337,12 @@ test.describe('when using the file tree to', () => {
       await test.step('swap between small and large files', async () => {
         await openDebugPanel()
         // Previously created a file so we need to start back at main.kcl
-        await mainFile.click()
+        await toolbar.openFile('main.kcl')
```

Comment:
`toolbar.openFile` does the `click` for me. I updated these lines to use the helper function.